src/main/java/com/example/music/backend/common/security/RestAuthenticationEntryPoint.java

```diff
@@ -0,0 +1,32 @@
+package com.example.music.backend.common.security;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.json.JSONObject;
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.security.web.AuthenticationEntryPoint;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class RestAuthenticationEntryPoint implements AuthenticationEntryPoint {
+
+    private ObjectMapper objectMapper = new ObjectMapper();
+
+    @Override
+    public void commence(
+            HttpServletRequest request,
+            HttpServletResponse response,
+            AuthenticationException exception)
+            throws IOException, ServletException {
+
+        response.setContentType("application/json;charset=UTF-8");
```

Comment:
Why didn't you use APPLICATION_JSON_UTF8_VALUE constant?

Reply:
replaced with constant APPLICATION_JSON_UTF8_VALUE